docs/README.md

```diff
@@ -34,6 +34,55 @@ You'll need a `kubectl`-style config file to connect to your cluster.
 `$HOME/.kube/config`. You can specify an alternate kubeconfig connection file
 with `--kubeconfig`, or the env var `$KUBECONFIG`, for any command.
 
+## Kn Config
+
+To better customize Kn there is a small set of configuration parameters you are able to setup. In particular you can specify where your Kn plugins are located and how they are found. The Kn configuration file is meant to capture these limited configuration options. Let's explore this file, it's location, and the current options you are able to change with it.
+
+### Location
+
+By default the Kn config file is located in the user's home directory under `.kn` hidden subdirectory and is named `config.yaml`. So in `~/.kn/config.yaml`. It is not created for you by default. You can easily create one with:
+
+```bash
+mkdir ~/.kn
+touch ~/.kn/config.yaml
+```
+
+Once that file exists (even when empty) Kn will indicate that is being used when you invoke a command. And default values for the configurable parameters will be added (if they were not present). For instance:
+
+```bash
+./kn plugin list
+Using kn config file: /home/maximilien/.kn/config.yaml
+No plugins found.
+```
+
+To see the values in the Kn config file you can do:
+
+```bash
+cat ~/.kn/config.yaml 
+lookuppluginsinpath: false
+pluginsdir: ~/.kn/plugins
+```
+
+You can now modify these values by editing the file. To specify a different config file then use the `--config` persistent flag when invoking Kn.
+
+### Options
+
+Currently there are two options you can specific in the Kn config file and they are related to how Kn locates plugins.
```

Comment:
remove "currently" please (or any other temporal expression as it is supposed to be a reference guide which always describes the current state of certain version)